### PR TITLE
fix: Update all source plugin to v0.12.2

### PR DESCRIPTION
**Pull request by yevgenypats**
#### Summary

also re-generate docs and add makefiles for every source plugin

### CODE REVIEW

File: plugins/source/aws/Makefile [hunk 1]
```diff
@@ -18,3 +18,8 @@ install-tools:
 .PHONY: install-hooks
 install-hooks:
 	pre-commit install
+
+.PHONY: gen-docs
+gen-docs:
+	go run main.go doc ./docs/tables
```
Comment (erezrokah) [inline plugins/source/aws/Makefile:24]: Maybe delete `docs/tables` here too (that's how our CI does it) to ensure removal of renamed tables

**Review by erezrokah — APPROVED** [review]

**Review by hermanschaaf — APPROVED** [review]

**Comment by yevgenypats** [comment]
@erezrokah do you know why workflows are not being run? (apart from aws)

File: .github/workflows/source_cloudflare.yml [hunk 2]
```diff
@@ -35,10 +35,8 @@ jobs:
...
         run: go build .
       - name: Test
-        run: go test ./...
-      - name: remove all docs
-        run: rm -f ./docs/tables/*.md
-      - name: build-docs
-        run: go run main.go doc docs/tables
+        run: make test
+      - name: gen-docs
+        run: go gen-docs
```
Comment (erezrokah) [inline .github/workflows/source_cloudflare.yml:40]:
```suggestion
        run: make gen-docs
```

**Comment by erezrokah** [comment]
@yevgenypats I would say this is either `fix:` or a `feat:`

**Review by erezrokah — APPROVED** [review]

**Pull request merged**